content/blog/2021-02-12-february-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,172 @@
+﻿---
+title: February '21 Heartbeat
+date: 2021-02-16
+description: |
+  Monthly updates are here! Read all about our growing team, 
+  our CEO's interview on The New Stack, integration with spaCy and more!
+descriptionLong: |
+  Monthly updates are here! Read all about our growing team, 
+  our CEO's interview on The New Stack, integration with spaCy and more!
+picture: 2021-02-11/feb21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/february-21-heartbeat/669
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - DAGsHub
+  - spaCy
+  - ML Summit 2021
+  - Spell
+  - MLOps
+---
+
+## News
+
+Happy February! Here's all the news to keep you up to date.
+
+## We've hired and are still hiring!
+
+We have four new team members this month!
+
+[**Dave Berenbaum**](https://www.linkedin.com/in/david-berenbaum-20b6b424/) came
+to Iterative.ai by way of a
+[previous contribution](https://github.com/iterative/dvc/pull/2107) to our open
+source products while working as a Data Science Manager at Captial One. He joins
+the team as a Technical Product Manager. We are thrilled he's here!
+
+[**Batuhan Taskaya**](https://www.linkedin.com/in/batuhan-osman-taskaya-7803b61a0/)
+joins us as a DVC Software Engineer working on the Python core. Batuhan is
+excited to work on open source full time and we are excited to have him do so!
+
+[**Jeny De Figueiredo**](https://www.linkedin.com/in/jenifer-de-figueiredo/) is
+involved in the Seattle area data science community at Data Circles and is a
+WiDS Puget Sound Ambassador. She joins us as our new Community Manager and is
+looking forward to further building and engaging the community in MLOps! (Hi!
+This is me. 🙋🏻‍♀️ I'll be writing Heartbeat!)
+
+[**Roger Parent**](https://www.linkedin.com/in/rogermparent/) has already been a
+big part of building DVC and [CML](https://cml.dev/). He has been a primary
+developer of a Visual Studio code extension that interfaces with the DVC Python
+application to provide an interface with the Experiments feature that's coming
+out with DVC 2.0. We are so excited to have him joining us full time as Software
+Engineer.
+
+![Alt Text](https://media.giphy.com/media/vAvWgk3NCFXTa/giphy.gif)
```

Comment:
💅 Alt Text - either put some real alt text, or remove it

Reply:
Done! 🎉